libs/designsystem/src/lib/components/calendar/calendar.component.spec.ts

```diff
@@ -454,4 +461,10 @@ describe('CalendarComponent', () => {
     spectator.output<Date>('dateSelect').subscribe((result) => (captured.event = result));
     return captured;
   }
+
+  function captureYearSelectEvents() {
+    const captured: { event?: number } = {};
+    spectator.output<number>('yearChange').subscribe((result) => (captured.event = result));
+    return captured;
+  }
```

Comment:
In the interest of keeping tests [isolated & flat](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test#the-good-test-is-isolated-and-flat), consider moving this directly to the test instead of having it in a separate function 😄

Reply:
Nevermind - I can see that the rest of the file does it like this as well. Disregard my comment, sorry!